themes/app/templates/Includes/Footer.ss

```diff
@@ -1,45 +1,45 @@
 <footer>
     <div class="container footer__partners">
         <div class="row">
-            <div class=" col-lg-2 col-md-3 col-sm-6 col-6 ">
+            <div class="col-lg-2 col-md-3 col-sm-6">
                 Our Partners
             </div>
         </div>
         <div class="row">
-            <div class=" col-lg-2 col-md-3 col-sm-6 col-6">
+            <div class="col-lg-2 col-md-3 col-sm-6 col-6">
```

Comment:
Ah my bad, should've made it clearer, remove the class `col-6` in all places from lines 9-24 :)

Reply:
Removed the col-sm instead and it works